proc/process.c

```diff
@@ -94,8 +95,13 @@ static void process_destroy(process_t *p)
 
 	posix_died(p->id, p->exit);
 
-	if (p->mapp != NULL)
+	if (p->mapp != NULL) {
 		vm_mapDestroy(p, p->mapp);
+	}
+
+	if (p->imapp != NULL) {
+		vm_mapDestroy(p, p->imapp);
```

Comment:
What do you think about destroying the `imapp` here, after `fork`:
https://github.com/phoenix-rtos/phoenix-rtos-kernel/blob/8b5a890117bd49e34c8c1cc750fdf7685ad23daa/proc/process.c#L1225

Reply:
Yes, thanks for spotting that